CHANGELOG.md

```diff
@@ -188,6 +188,452 @@
 
 
 
+## 47.9.1 (2022-11-04)
```

Comment:
I would revert all the changes to the Changelog, a new change will be generated when you merge your PR